composer.json

```diff
@@ -1,23 +1,43 @@
 {
     "name": "micro/dependency-injection",
     "description": "Dependency injection service container",
-    "type": "library",
     "license": "MIT",
-    "version": "1.1",
-    "autoload": {
-        "psr-4": {
-            "Micro\\Component\\DependencyInjection\\": "src/"
-        }
-    },
+    "type": "library",
     "authors": [
         {
             "name": "Stanislau.Komar",
             "email": "stanislau_komar@epam.com"
         }
     ],
     "require": {
-        "php": ">=8.0",
-        "psr/container": "^2.0",
-        "micro/autowire": "^1"
+        "php": "^8.1 || ^8.2",
```

Comment:
php version should be equal 8.2 or higher.

Reply:
@Asisyas, done